src/components/Divider.jsx

```diff
@@ -0,0 +1,25 @@
+import { Typography } from '@mui/material'
+import React from 'react'
+
+export default function Divider(props) {
+    const lineStyle = {
+        width: '40%',
+        height: '1px',
+        backgroundColor: '#854D27'
+    }
+
+    const flex = {
```

Comment:
Okay, so try to follow one standard,
Lets do one thing, lets review this together in the coming meeting session.
We should either use styled components or class names or inline styles.
Never mix all of them together :D